config/initializers/wings.rb

```diff
@@ -110,34 +105,8 @@
   end
 
   Valkyrie.config.resource_class_resolver = lambda do |resource_klass_name|
-    # TODO: Can we use some kind of lookup.
-    klass_name = resource_klass_name.gsub(/Resource$/, '')
-    if %w[
-      GenericWork
-      Image
-      Etd
-      Oer
-    ].include?(klass_name)
-      "#{klass_name}Resource".constantize
-    elsif 'Collection' == klass_name
-      CollectionResource
-    elsif 'AdminSet' == klass_name
-      AdminSetResource
-    # Without this mapping, we'll see cases of Postgres Valkyrie adapter attempting to write to
-    # Fedora.  Yeah!
-    elsif 'Hydra::AccessControl' == klass_name
-      Hyrax::AccessControl
-    elsif 'FileSet' == klass_name
-      Hyrax::FileSet
-    elsif 'Hydra::AccessControls::Embargo' == klass_name
-      Hyrax::Embargo
-    elsif 'Hydra::AccessControls::Lease' == klass_name
-      Hyrax::Lease
-    elsif 'Hydra::PCDM::File' == klass_name
-      Hyrax::FileMetadata
-    else
```

Comment:
I'm a bit concerned with the removal of the above mappings. Where does Hydra::AccessControls::Embargo get mapped to Hyrax::Embargo, etc? All of these were added due to prior errors, and I want to be sure we aren't breaking prior fixes.

Reply:
They are initialized in Hyrax wings adapter. If you open the Hyku console and do Wings::ModelRegistry.reverse_lookup(Hydra::AccessControls::Embargo) you will see the right result

Reply:
Or at least I did when I tried them all one by one

Reply:
I tried a few and they worked so I approved.